.github/actions/setup-minikube-cluster/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
       # See https://github.com/actions/runner/issues/834
       uses: ChristopherHX/conditional@b4a9649204f81002ec9a4ef7d4bf7d6b2ab7fa55
       with:
-        if: ${{ inputs.k8s_version != '' }}
+        if: ${{ inputs.k8s_version != '' && inputs.k8s_version != 'v1.24.0' }}
```

Comment:
Why disallow this k8s version?

Reply:
nvm. I see your comment: https://github.com/scalyr/scalyr-agent-2/pull/885#issuecomment-1132657663

Reply:
We don't set "start args" action parameter for v1.24.0 (see step below).